client/wp-admin-scripts/onboarding-homepage-notice/index.js

```diff
@@ -77,8 +82,11 @@ const onboardingHomepageNotice = () => {
 				type: notificationType,
 				actions: [
 					{
-						url: getAdminLink( 'admin.php?page=wc-admin&task=appearance' ),
 						label: __( 'Continue setup.', 'woocommerce-admin' ),
+						onClick: () => {
+							recordEvent( 'tasklist_appearance_continue_setup', {} );
```

Comment:
Probably not an issue, but just curious if `recordEvent` is async.  Is there any chance this request has not completed before we change pages?

Looks like it might be using `ga` under the hood and I think this is okay to fire events on page change, but just want to verify.

Reply:
The closest answer I could find for this was in the Calypso docs https://github.com/Automattic/wp-calypso/blob/a5c00596e329b79f185803328c3af8caade233bb/client/lib/analytics/README.md#what-is-the-analytics-queue which I think use the same underlying `stats.wp.com/w.js` script -- and it looks like we could run into issues here.

Since we don't currently have an analytics queue (which might be something to consider for WooCommerce Admin), I ended up moving this check to the next pageload of appearance -- do you think this works?

https://github.com/woocommerce/woocommerce-admin/pull/3220/commits/51bbbf95868ec1004b2137ad443e968da73fa212